rotkehlchen/api/server.py

```diff
@@ -236,6 +237,7 @@
         'per_report_data_resource',
     ),
     ('/accounting/rules', AccountingRulesResource),
+    ('/accounting/rules/transfer', AccountingRulesTransferResource),
```

Comment:
Change name here and elsewhere. Create two different endpoints. One import and one export. Since /transfer is a really ambiguous name and does not convey what is happening here which is import or export.

```suggestion
    ('/accounting/rules', AccountingRulesTransferResource),
```